docs/usage/modularizing.md

```diff
@@ -0,0 +1,77 @@
+
+(modularizing-python-codes)=
+# Modularizing Python codes
```

Comment:
Maybe "Packaging Python code for Pyodide"?

Reply:
Yes, that seems better.

Reply:
Hmm... but someone might be confused about the difference between "Creating a Pyodide package" and "Packaging Python code for Pyodide"

Reply:
Bundling Python code for Pyodide?

Reply:
Or `Loading custom Python code` since I guess most of this is going to be about that, and this should be separate from bundling code in packages  ?

Reply:
I'll go with `Loading custom Python code`